### DISCUSSION TITLE
How to start with preview window already minimised?

### DISCUSSION THREAD

**Discussion by faxotherapy** [junegunn/fzf#3274] [Q&A] [answered]
I have set `--preview 'glow -p -w 109 -s dark {}' --preview-window='bottom,66%,wrap' --bind 'ctrl-/:toggle-preview'` for `FZF_CTRL_T_OPTS`. My problem is that fzf always starts with the preview window wide and large. Is it possible to minimise it at start?

**Comment by Ioulan** [comment, marked as answer]
excerpted from the man page on v39
 ```
--preview-window=[POSITION][,SIZE[%]][,border-BOR‐
       DER_OPT][,[no]wrap][,[no]follow][,[no]cycle][,[no]hidden][,+SCROLL[OFF‐
       SETS][/DENOM]][,~HEADER_LINES][,default][,<SIZE_THRESHOLD(ALTERNATIVE_LAY‐
       OUT)]
```


`--preview-window=hidden`

On v30 above 
You can also adjust the preview on a certain threshold 

`--preview-window '<50(nohidden,border-bottom)'`
